components/tools/OmeroWeb/omeroweb/webgateway/static/webgateway/js/ome.viewport.js

```diff
@@ -603,9 +603,9 @@ jQuery._WeblitzViewport = function (container, server, options) {
     this.setChannelActive(idx, !_this.loadedImg.channels[idx].active);
   };
 
-  this.setChannelReverseIntensity = function (idx, reverse, noreload) {
```

Comment:
Removal of this function is another breaking change. I assume the cost of having it as an alias is low enough that it would be worth it.

```
this. setChannelReverseIntensity = this. setChannelInverted
```